Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: run_all run watch test clean prod_build deps dev_db fmt
 
-# Runs the application.
+# Runs the application (builds Vue.js files, starts Next.js dev server, starts Go server).
 run_all:
-	cd frontend && npm run dev-build
+	. ~/.nvm/nvm.sh && \
+    (cd frontend && nvm install 16 && npm run dev-build) && \
```

Comment:
This is kinda annoying. But we have some super old deps in the Vue app that break if we go past Node 16. But we need at least 18 to use the latest React.

Reply:
that would be useful to add as a comment

// As of January 2025, upgrading past Node 16 breaks old Vue dependencies, and Node 18 is required to use the latest version of React.